.ci/scripts/update-beats.sh

```diff
@@ -11,7 +11,8 @@ COMMIT_MESSAGE="Update to elastic/beats@$(go list -m -f {{.Version}} github.com/
 git checkout -b "update-beats-$(date "+%Y%m%d%H%M%S")"
 git add go.mod go.sum NOTICE.txt \
 	.go-version docs/version.asciidoc \
-	docs/fields.asciidoc include/fields.go x-pack/apm-server/include/fields.go
+	docs/fields.asciidoc include/fields.go x-pack/apm-server/include/fields.go || true
```

Comment:
We should probably use the `--ignore-errors` option for `git add` instead of `|| true`

Reply:
I like that one! I just reviewed it's supported since https://git-scm.com/docs/git-add/2.3.10 or even earlier. So we are safe to use it